src/libs/actions/IOU.ts

```diff
@@ -3103,7 +3073,7 @@ function getSendMoneyParams(report, amount, currency, comment, paymentMethodType
             paymentMethodType,
             transactionID: optimisticTransaction.transactionID,
             newIOUReportDetails,
-            createdReportActionID: isNewChat ? optimisticCreatedAction.reportActionID : 0,
+            createdReportActionID: isNewChat ? optimisticCreatedAction.reportActionID : '',
```

Comment:
This created a warning on the server, polluting the logs. It should be '0' instead of an empty string, as the server expects an integer. [Here](https://github.com/Expensify/App/pull/35918)'s a PR that fixes it

Reply:
Got it, thank you!